tests/model_serving/model_runtime/model_validation/conftest.py

```
@@ -0,0 +1,200 @@
+@pytest.fixture(scope="function")
+def modelcar_image_uri(request: FixtureRequest, model_image_name: str | list[str], registry_host: str) -> str:
+    """
+    Returns the model image URI for the modelcar image.
+    If the model image name is not provided, it skips the test.
+    """
+    param_index = request.node.callspec.indices.get("modelcar_image_uri", None)
+    if param_index is not None and param_index < len(model_image_name):
+        override = model_image_name[param_index]
+    else:
+        pytest.skip("model_image_name completed, no more parameters to test.")
+    return f"oci://{registry_host}/rhelai1/{override}"
```

Pylint (3.3.7)

[error] Possibly using variable 'override' before assignment

(E0606)